cmd/client.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
you should have another structure for the project:

cmd:
--client:
----main.go
--server:
----main.go

Reply:
Resolved in https://github.com/m-kuzmin/golang-tcp-client-and-server/pull/1/commits/4717acc9a8c2f3911616dd67ee3f5773f20e3dcb